lbl/science/general.py

```diff
@@ -1265,16 +1298,21 @@ def compute_rv(inst: InstrumentsType, sci_iteration: int,
                     for key in inst.params['RESPROJ_TABLES']:
                         # calculate spline
                         rp_spline = splines[key](wave_ord)
-                        # multiply by blaze and ratio
-                        rp_spline = rp_spline * blaze_ord * ratio[order_num]
+                        # The models are always expressed in terms of the
+                        # original spectrum
+                        rblaze = np.nanmedian(sci_data0[order_num]/blaze_ord)
+                        rp_spline *= (blaze_ord * rblaze)
                         # add to projection model
                         proj_model[key]['model'][order_num] = rp_spline
         # ---------------------------------------------------------------------
         # estimate rms
         # ---------------------------------------------------------------------
         # if we are not using a noise model - estimate the noise
         if not use_noise_model:
-            rms = estimate_noise_model(sci_data, wavegrid, model, hp_width)
+            # TODO -> noise_sampling_width needs to be in the config file
+            noise_sampling_width = 200000 # 200 km/s
```

Comment:
could you try putting this in the config yourself?

Step 1: add it to the relevant section of `lbl/core/parameters.py` you can add 200000 as the default value only if it is valid for all instruments.

Step 2: If it isn't 200000 for all instruments you need to add it to all the instrument profiles in `lbl/instruments/`, make sure you don't miss one

I think you need to be more specific than `noise_sampling_width`  maybe `NOISE_MODEL_SAMPLING_WIDTH` just so it isn't changed by anyone else.